tests/foreman/ui/test_template.py

```diff
@@ -155,10 +155,10 @@ def test_negative_create_without_upload(self):
                     custom_really=True,
                     template_type='PXELinux template',
                 )
-                self.assertEqual(
-                    context.exception.message,
-                    'Could not create blank template "{0}"'.format(name)
-                )
+            self.assertEqual(
+                context.exception.message,
+                'Could not create blank template "{0}"'.format(name)
```

Comment:
prefix the formatting string with `u` name may contain unicode characters and that will fail on Python 2.